nselib/data/http-fingerprints.lua

```diff
@@ -12562,6 +12562,119 @@ table.insert(fingerprints, {
     },
   });
 
+-- Atlassian Jira
+table.insert(fingerprints, {
+  category = 'atlassian',
+  probes = {
+    {
+      path = '/rest/applinks/1.0/manifest',
+      method = 'GET'
+    },
+    {
+      path = '/jira/rest/applinks/1.0/manifest',
+      method = 'GET'
+    },
+    {
+      path = '/secure/rest/applinks/1.0/manifest',
+      method = 'GET'
+    },
+  },
+  matches = {
+    {
+      match = '<typeId>jira</typeId>.*<version>(.*)</version>',
```

Comment:
To avoid excessive backtracking (very slow pattern matching), can you rewrite this pattern to be more strict and not use 2 "match anything" patterns (`.*`)? Easiest would probably be changing the second one to `[^<]+` to make sure it matches at least one character and does not match another XML tag, but it depends on what you expect the data to be. Thanks!

Reply:
Thanks @dmiller-nmap!

I went ahead and made the changes that. you requested